### PR TITLE
Feature/31624 add maven plugin

### PR DESCRIPTION
**Pull request by christiangebhard**
for configuration of local credentials check https://dev.azure.com/evoila/KuSIM/_wiki/wikis/KuSIM.wiki/589/Helm-Wrapper?anchor=fabric8io/docker-maven-plugin

### CODE REVIEW

**Review by felsta — APPROVED** [review]

File: version [hunk 1]
```diff
@@ -1 +0,0 @@
-0.1.3-SNAPSHOT
```
Comment (miriSch) [inline version:1]: how was this used before? did the helm-wrapper executable know its own version?

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <properties>
+        <registry.imagePath>harbor.evoila-lab.work/bu-dev/kusim/images</registry.imagePath>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>io.fabric8</groupId>
+                <artifactId>docker-maven-plugin</artifactId>
+                <extensions>true</extensions>
```
Comment (miriSch) [inline pom.xml]: is this line necessary? documentation sounds like it is needed for custom lifecycles

File: pom.xml
Comment (miriSch) [inline pom.xml:1]: question for future integration into an aggregator: Does `mvn clean verify docker:build` also run successfully?

File: version (hunk 1 above)
Comment (christiangebhard) [reply version:1]: This file was added by Philipp, https://github.com/evoila/helm-wrapper/commit/25dd956bbd4bef8fd6820cedc6594a59178ed123 in order to set image versions in the pipelines

File: pom.xml (hunk 2 above)
Comment (christiangebhard) [reply pom.xml]: I will check again, from the examples I checked it was there every time (even for simple ones like this)

File: pom.xml
Comment (christiangebhard) [reply pom.xml:1]: yes runs successfully

**Review by niksteph — APPROVED** [review]

**Review by miriSch — APPROVED** [review]

**Pull request merged**